repository-commons/build.gradle

```diff
@@ -15,7 +15,7 @@
  */
 
 ext {
-    zstdVersion = "1.4.5-6"
+    zstdVersion = "1.4.5-12"
```

Comment:
Is it necessary in this commit/PR?

Reply:
Yes ZSTD now uses the buffer pool for compression/decropression instead of creation all the time a new buffer. Should reduce usage of CPU

Reply:
@ivanyu exluded ZSTD new version, will prepare separate PR